src/docker-compose.dcproj

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" Sdk="Microsoft.Docker.Sdk">
+  <PropertyGroup Label="Globals">
+    <ProjectVersion>2.1</ProjectVersion>
+    <DockerTargetOS>Linux</DockerTargetOS>
+    <ProjectGuid>f5df2fdc-c860-4cb3-8b24-7c903c6fc076</ProjectGuid>
+  </PropertyGroup>
+  <PropertyGroup>
+    <DockerServiceName>microsoft.data.sqlclient.dockertests</DockerServiceName>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="docker-compose.override.yml">
+      <DependentUpon>docker-compose.yml</DependentUpon>
+    </None>
+    <None Include="docker-compose.yml" />
+    <None Include=".dockerignore" />
+  </ItemGroup>
+</Project>
```

Comment:
Will adding this project type change any requirements in our BUILDGUIDE? (I don't have Docker installed, for example.)

Reply:
Building & Testing should be fine but for seamless experience in Visual Studio I can update the vsconfig files to include docker too.

Reply:
Yes, do not expect everyone to have Docker Tools installed

Reply:
I verified and our VS19Components.vsconfig includes "Container Tools" as tested on v16.5.1.